_inc/client/components/masthead/style.scss

```diff
@@ -1,25 +1,29 @@
 .jp-masthead {
 	background-color: $green-secondary;
 	text-align: center;
+
+	@media (max-width: rem( 720px ) ) {
```

Comment:
Spacing before/after `(`? Not sure exactly what standard we're following on CSS, but this doesn't look right :)